### PR TITLE
Increase lint coverage

### PR DESCRIPTION
**Pull request by vaporos**

### CODE REVIEW

**Comment by vaporos** [comment]
The first commit here has been modified from the version in #109, with the two commits immediately after it resolving the a few of the linting issues in a slightly different manner.

File: validator/sawtooth_validator/journal/consensus/poet1/poet_transaction_block.py [hunk 1]
```diff
@@ -158,7 +165,7 @@ def update_block_weight(self, journal):
             assert self.wait_certificate
             self.aggregate_local_mean = self.wait_certificate.local_mean
 
-            if self.PreviousBlockID != NullIdentifier:
+            if self.PreviousBlockID != NULL_BLOCK_IDENTIFIER:
```
Comment (peterschwarz) [inline validator/sawtooth_validator/journal/consensus/poet1/poet_transaction_block.py]: Any reason why this doesn't need to be lower-cased?

File: validator/sawtooth_validator/journal/consensus/poet1/wait_certificate.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 # ------------------------------------------------------------------------------
 
 import logging
-from requests import ConnectionError
+from requests import ConnectionError as RequestsConnectionError
```
Comment (peterschwarz) [inline validator/sawtooth_validator/journal/consensus/poet1/wait_certificate.py]: Instead renaming, just reference the exception via `requests.ConnectionError`

**Comment by vaporos** [comment]
Commit d07a6ed is broken.  I'll work on a replacement.

Also, cool that the smoke test caught it.

File: validator/sawtooth_validator/journal/consensus/poet1/poet_transaction_block.py (hunk 1 above)
Comment (peterschwarz) [inline validator/sawtooth_validator/journal/consensus/poet1/poet_transaction_block.py]: And by "this" I mean `PreviousBlockId`

Comment (vaporos) [inline validator/sawtooth_validator/journal/consensus/poet1/poet_transaction_block.py]: It declared at the module level (similar to LOGGER), and pep8 specifies those follow all-caps.

Comment (peterschwarz) [inline validator/sawtooth_validator/journal/consensus/poet1/poet_transaction_block.py]: See my previous reply - I meant `PreviousBuildID`, not `NULL_BLOCK_IDENTIFIER`

Comment (cintel) [inline validator/sawtooth_validator/journal/consensus/poet1/poet_transaction_block.py]: PreviousBlockID will be converted to lower case when this module is converted to use the 0.8 protobuf block definition.

File: validator/sawtooth_validator/journal/consensus/poet1/wait_certificate.py (hunk 2 above)
Comment (vaporos) [inline validator/sawtooth_validator/journal/consensus/poet1/wait_certificate.py]: This has been updated.

**Comment by vaporos** [comment]
The super.__init__() build issue has been fixed.

**Pull request merged**